jme3-core/src/main/java/com/jme3/material/UnshadedMaterial.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package com.jme3.material;
+
+import com.jme3.asset.AssetManager;
+import com.jme3.math.ColorRGBA;
+import com.jme3.texture.Texture;
+
+/**
+ * Material using Unshaded.j3md.
+ * 
+ * @author codex
+ */
+public class UnshadedMaterial extends Material {
+    
+    /**
+     * Creates a material with Unshaded.j3md.
+     * 
+     * @param assetManager 
+     */
+    public UnshadedMaterial(AssetManager assetManager) {
+        super(assetManager, "Common/MatDefs/Misc/Unshaded.j3md");
```

Comment:
Note the matdef asset paths are defined as constants in `com.jme3.material.Materials` .
Perhaps these new classes should make use of the existing constants.